README.md

```diff
@@ -253,6 +242,67 @@ clientConfig.getNetworkConfig().getKubernetesConfig().setEnabled(true)
             .setProperty("service-name", "MY-SERVICE-NAME");
 ```
 
+### Outside Kubernetes Cluster
+
```

Comment:
I would add what it means `Outside Kubernetes Cluster` before explaining what to do with dummy/smart client cases... like cluster only reachable through public internet etc.

Reply:
Ok, added one sentence on that.